thirdeye/thirdeye-frontend/app/pods/components/filter-bar/component.js

```diff
@@ -81,7 +77,8 @@ export default Ember.Component.extend({
       });
 
       // Now add new initialized props to block item
-      Object.assign(block, { filtersArray, isHidden });
+      Ember.set(block, 'filtersArray', filtersArray);
+      Ember.set(block, 'isHidden', isHidden);
```

Comment:
I believe the following should work here
```
Ember.setProperties(block, { 
  filtersArray,
  isHidden
})
```

Reply:
when we use `this.set` and `this.setProperties` the context is obvious (this), but when using Ember.set, Ember.setProperties, the first arguments becomes the context.